extra_script.py

```diff
@@ -1,20 +1,21 @@
 Import("env")
 import os, sys
-import microros_utils.library_builder as library_builder
 
 ##############################
 #### Install dependencies ####
 ##############################
 
 pip_packages = [x.split("==")[0] for x in os.popen('{} -m pip freeze'.format(env['PYTHONEXE'])).read().split('\n')]
-required_packages = ["catkin-pkg", "lark-parser", "empy", "colcon-common-extensions", "importlib-resources"]
+required_packages = ["catkin-pkg", "lark-parser", "empy", "colcon-common-extensions", "importlib-resources", "pyyaml"]
 if all([x in pip_packages for x in required_packages]):
     print("All required Python pip packages are installed")
 
 for p in [x for x in required_packages if x not in pip_packages]:
     print('Installing {} with pip at PlatformIO environment'.format(p))
     env.Execute('$PYTHONEXE -m pip install {}'.format(p))
 
+import microros_utils.library_builder as library_builder
```

Comment:
Why are we moving this?

Reply:
Because `pyyaml` is used on `library_builder`, but its installed on this previous lines. 

In summary: To import `library_builder`, `required_packages` should be already installed.

Reply:
ok